python/desc/sims/GCRCatSimInterface/om10_lensing_equations.py

```diff
@@ -0,0 +1,168 @@
+import os
+import numpy as np
+import pylab as pl
+from scipy.interpolate import interp1d
+#--------------------------------------------------------------------
+from astropy.cosmology import Planck15 as p15
+
+vc = 2.998e5 #km/s
+G = 4.3011790220362e-09 # Mpc/h (Msun/h)^-1 (km/s)^2
+apr =  206269.43        # 1/1^{''}
+
+data_dir = os.path.join(os.path.abspath("../../../../"),'data')
+
+def Dc(z):
+    res = p15.comoving_distance(z).value*p15.h
```

Comment:
I don't think we are actually using the Planck cosmology for the extragalactic catalogs.  The GCR allows you to get access to the cosmology we are using from something like
```
cat = GCRCatalogs.load_catalogs(yaml_file)
cosmology = cat.cosomology
```
(see, for instance
https://github.com/LSSTDESC/sims_GCRCatSimInterface/blob/master/python/desc/sims/GCRCatSimInterface/CatalogClasses.py#L297 )

You can get access to the GCR catalo though any of the `*DESCQAObject`s instantiated in the `InstanceCatalogWriter`, for example, here

https://github.com/LSSTDESC/sims_GCRCatSimInterface/blob/master/python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py#L274